src/shared/components/unknownStudies/UnknownStudiesWarning.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
```

Comment:
you should tag this with @observer decorator so that it will react to changes in observable properties (ids).  for example, the session might change, in which case the warning should go away.